docs/standard-library/variant.md

```diff
@@ -26,7 +26,7 @@ A variant object holds and manages a value. If the variant holds a value, that v
 |[operator<](../standard-library/forward-list-operators.md#op_lt)|Tests if the variant object on the left side of the operator is less than the variant object on the right side.|
 |[operator<=](../standard-library/forward-list-operators.md#op_lt_eq)|Tests if the variant object on the left side of the operator is less than or equal to the variant object on the right side.|
 |[operator>](../standard-library/forward-list-operators.md#op_gt)|Tests if the variant object on the left side of the operator is greater than the variant object on the right side.|
-|[operator>=](../standard-library/forward-list-operators.md#op_lt_eq)|Tests if the variant object on the left side of the operator is greater than or equal to the variant object on the right side.|
+|[operator>=](../standard-library/forward-list-operators.md#op_gt_eq)|Tests if the variant object on the left side of the operator is greater than or equal to the variant object on the right side.|
```

Comment:
All of the above links should point to [`docs/standard-library/variant-operators.md`](https://github.com/MicrosoftDocs/cpp-docs/blob/45fdb38aa4947fc6646b7727e816e136e636941b/docs/standard-library/variant-operators.md) instead. A separate PR will be issued once this is merged.